server/command.go

```diff
@@ -174,6 +174,84 @@ func (p *Plugin) executeCommandQueue(args *model.CommandArgs) *model.CommandResp
 	return &model.CommandResponse{}
 }
 
+func calculateQueItemNumber(args *model.CommandArgs, p *Plugin, hashtag string) (*model.CommandResponse, int) {
+	searchResults, appErr := p.API.SearchPostsInTeamForUser(args.TeamId, args.UserId, model.SearchParameter{Terms: &hashtag})
+	if appErr != nil {
+		return responsef("Error calculating list number"), 0
+	}
+	postList := *searchResults.PostList
+	numQueueItems := len(postList.Posts)
+	return nil, numQueueItems + 1
+}
+
+func (p *Plugin) executeCommandReQueue(args *model.CommandArgs) *model.CommandResponse {
```

Comment:
There's a lot going on in this function. Can we pull some of the logic out to be unit tested?

Reply:
It doesn't look like this function was reactored, but we agreed it should be refactored and tested in this conversation